ui/app/components/job-status/panel/steady.hbs

```diff
@@ -48,7 +48,7 @@
               @condition={{not (eq type.label "unplaced")}}
               @route="jobs.job.allocations"
               @model={{@job}}
-              @query={{hash status=(concat '["' type.label '"]') version=(concat '[' (keys this.versions) ']')}}
+              @query={{hash status=(concat '["' type.label '"]') version=(concat '[' (map-by "version" this.versions) ']')}}
```

Comment:
Noted that I was being inconsistent after changing my version sorting logic a few PRs ago.